Last-Update: 2012-08-24
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Forwarded: not-needed
Description: backported patch (r31576)
 Remove separator from Insert dialogs (GTK2 only).

--- a/src/wp/ap/gtk/ap_UnixDialog_InsertBookmark.cpp
+++ b/src/wp/ap/gtk/ap_UnixDialog_InsertBookmark.cpp
@@ -187,6 +187,9 @@
   pSS->getValueUTF8(AP_STRING_ID_DLG_InsertBookmark_Title,s);
   
   m_windowMain = abiDialogNew("insert bookmark dialog", TRUE, s.utf8_str());
+#if !GTK_CHECK_VERSION(3,0,0)
+  gtk_dialog_set_has_separator(GTK_DIALOG(m_windowMain), FALSE);
+#endif
 
   
   vbox = gtk_box_new (GTK_ORIENTATION_VERTICAL, 6);
--- a/src/wp/ap/gtk/ap_UnixDialog_InsertHyperlink.cpp
+++ b/src/wp/ap/gtk/ap_UnixDialog_InsertHyperlink.cpp
@@ -205,6 +205,9 @@
   UT_UTF8String s;
   pSS->getValueUTF8(AP_STRING_ID_DLG_InsertHyperlink_Title,s);
   m_windowMain = abiDialogNew("insert table dialog", TRUE, s.utf8_str());
+#if !GTK_CHECK_VERSION(3,0,0)
+  gtk_dialog_set_has_separator(GTK_DIALOG(m_windowMain), FALSE);
+#endif
 
   frame1 = gtk_frame_new (NULL);
   gtk_widget_show (frame1);
--- a/src/wp/ap/gtk/ap_UnixDialog_InsertXMLID.cpp
+++ b/src/wp/ap/gtk/ap_UnixDialog_InsertXMLID.cpp
@@ -223,6 +223,9 @@
     pSS->getValueUTF8(AP_STRING_ID_DLG_InsertXMLID_Title,s);
   
     m_window = abiDialogNew("insert RDF link dialog", TRUE, s.utf8_str());
+#if !GTK_CHECK_VERSION(3,0,0)
+    gtk_dialog_set_has_separator(GTK_DIALOG(m_window), FALSE);
+#endif
 
   
     vbox = gtk_box_new (GTK_ORIENTATION_VERTICAL, 6);
